presto-main/src/main/java/com/facebook/presto/server/SerializedPageWriteListener.java

```diff
@@ -78,6 +79,8 @@ public void onWritePossible()
                 bufferPosition += SIZE_OF_INT;
                 slice.setInt(bufferPosition, page.getSizeInBytes());
                 bufferPosition += SIZE_OF_INT;
+                slice.setLong(bufferPosition, page.getChecksum());
```

Comment:
@arhimondr this is what got missed in the previous PR. Wondering if we can reuse the logic from PageSerdeUril.writeSerializedPage

Reply:
Yeah, let me refactor it